javascript/javascript3/week2/school_database.sql

```diff
@@ -0,0 +1,65 @@
+
+/* Part 2: School database */
+/* 1. Create a new database containing the following tables: */
+/*   - Class: with the columns: id, name, begins (date), ends (date) */
+/*   - Student: with the columns: id, name, email, phone, class_id (foreign key) */
+/* 2. If you are done with the above tasks, you can continue with these advanced tasks: */
+/*   - Create an index on the name column of the student table. */
+/*   - Add a new column to the class table named status which can only have */
+/* the following values: not-started, ongoing, finished (hint: enumerations). */
+
+
+/* Setup school database */
+CREATE DATABASE school_database;
+Use school_database;
+
+/* Create class table */
+CREATE TABLE `class` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `name` varchar(255) NOT NULL,
+  `begins` DATETIME NOT NULL,
+  `ends` DATETIME NOT NULL
+);
+
+/* Create student table */
+CREATE TABLE `student` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `name` varchar(255) NOT NULL,
+  `email` varchar(255) NOT NULL,
+  `phone` varchar(255) NULL,
+  `class_id` int(10) UNSIGNED NOT NULL,
+  CONSTRAINT `fk_class` FOREIGN KEY (`class_id`) REFERENCES `class` (`id`) ON DELETE CASCADE
+);
+
+/* Add class data */
+INSERT INTO class (id, name, begins, ends)
+VALUES (1, "Class01", "2020-08-01", "2020-10-01");
```

Comment:
If the `begins` and `ends`  fields are `DATETIME` then you should be consistent with that and insert a `DATETIME`  here as well.